awx/main/notifications/webhook_backend.py

```diff
@@ -16,13 +17,17 @@ class WebhookBackend(AWXBaseEmailBackend):
 
     init_parameters = {"url": {"label": "Target URL", "type": "string"},
                        "disable_ssl_verification": {"label": "Verify SSL", "type": "bool", "default": False},
+                       "username": {"label": "Username", "type": "string", "default": ""},
```

Comment:
Are `default` values necessary here?  It doesn't seem like we have this in other places.

Reply:
I couldn't make `username` and `password` optional fields without inputting the default of an empty string (@jladdjr and @rooftopcellist witnessed my struggle with making this not be a required parameter).  Lots of things were tried out, this seemed to do the trick.